cypress/e2e/api/UsersApi.spec.js

```diff
@@ -0,0 +1,110 @@
+/*
+ * @copyright Copyright (c) 2022 Max <max@nextcloud.com>
+ *
+ * @author Max <max@nextcloud.com>
```

Comment:
I assume this is your copyright - or has this test only been moved?

Reply:
I started with a copy of the other API one ;) Will update to keep us both